Gemfile.lock

```diff
@@ -648,4 +671,4 @@ DEPENDENCIES
   xmpp4r (~> 0.5.6)
 
 BUNDLED WITH
-   1.10.6
+   1.11.2
```

Comment:
Will this affect people with older Bundler versions?  I'm not entirely clear on how "BUNDLED WITH" works or what it does.